### PR TITLE
Add Elm Tailwind CSS community template

### PR DESCRIPTION
**Pull request by rnice01**
This template is for folks looking to use Elm and Tailwind CSS to develop their Wails app. Supports hot reloading 🎉

### CODE REVIEW

File: website/docs/community/templates.mdx [hunk 1]
```diff
@@ -48,6 +48,7 @@ If you are unsure about a template, inspect `package.json` and `wails.json` for
 ## Elm
 
 - [wails-elm-template](https://github.com/benjamin-thomas/wails-elm-template) - Develop your GUI app with functional programming and a **snappy** hot-reload setup :tada: :rocket:
+- [wails-elm-tailwind-template](https://github.com/rnice01/wails-elm-tailwind-template) - Combine the powers :muscle: of Elm + Tailwind CSS + Wails! Hot reloading supported.
```
Comment (misitebao) [inline website/docs/community/templates.mdx:51]: It is recommended to use the naming format of `wails-template-xxx`

Comment (rnice01) [reply website/docs/community/templates.mdx:51]: Did you want me to just update the link text or rename the whole repo?

Comment (misitebao) [reply website/docs/community/templates.mdx:51]: Maybe both, thanks.

Comment (leaanthony) [reply website/docs/community/templates.mdx:51]: @rnice01 - Any updates? Would *love* to get this in 👍

Comment (rnice01) [reply website/docs/community/templates.mdx]: @misitebao @leaanthony 
It's been updated, thanks for your patience!

**Review by misitebao — COMMENTED** [review]
LGTM.

**Review by misitebao — COMMENTED** [review]
LGTM.

**Comment by leaanthony** [comment]
Thanks for this!!! 🙏

**Pull request merged**